Cant compile the ndk-context example from the documentation

I have tried compiling the example in the docs

```rust
let ctx = ndk_context::android_context();
        let vm = unsafe { jni::JavaVM::from_raw(ctx.vm().cast()) }.unwrap();
        let env = vm.attach_current_thread().unwrap();
        let class_ctx = env.find_class("android/content/Context").unwrap();
        let audio_service = env
            .get_static_field(class_ctx, "AUDIO_SERVICE", "Ljava/lang/String;")
            .unwrap();
        let audio_manager = env
            .call_method(
                ctx.context() as jni::sys::jobject,
                "getSystemService",
                "(Ljava/lang/String;)Ljava/lang/Object;",
                &[audio_service],
            )
            .unwrap()
            .l()
            .unwrap();
```

I get the error

```sh
cargo build --target x86_64-linux-android 
   Compiling notes_native v0.1.0 (/home/test1/Notes/notes/NotesNative)
warning: unused import: `jni::*`
 --> NotesNative/src/app.rs:2:5
  |
2 | use jni::*;
  |     ^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

error[E0277]: the trait bound `JObject<'_>: From<*mut _>` is not satisfied
   --> NotesNative/src/app.rs:42:13
    |
41  |         env.call_method(
    |             ----------- required by a bound introduced by this call
42  |             ctx.context().cast(),
    |             ^^^^^^^^^^^^^^^^^^^^ the trait `From<*mut _>` is not implemented for `JObject<'_>`
    |
    = help: the following other types implement trait `From<T>`:
              <JObject<'a> as From<&'a AutoLocal<'a, '_>>>
              <JObject<'a> as From<&'a GlobalRef>>
              <JObject<'a> as From<JByteBuffer<'a>>>
              <JObject<'a> as From<JClass<'a>>>
              <JObject<'a> as From<JList<'a, 'b>>>
              <JObject<'a> as From<JMap<'a, 'b>>>
              <JObject<'a> as From<JString<'a>>>
              <JObject<'a> as From<JThrowable<'a>>>
    = note: required for `*mut _` to implement `Into<JObject<'_>>`
note: required by a bound in `JNIEnv::<'a>::call_method`
   --> /home/test1/.cargo/registry/src/github.com-1ecc6299db9ec823/jni-0.20.0/src/wrapper/jnienv.rs:988:12
    |
988 |         O: Into<JObject<'a>>,
    |            ^^^^^^^^^^^^^^^^^ required by this bound in `JNIEnv::<'a>::call_method`

For more information about this error, try `rustc --explain E0277`.
warning: `notes_native` (lib) generated 1 warning
error: could not compile `notes_native` due to previous error; 1 warning emitted
```


Where could I be wrong ?

--- a/ndk-examples/Cargo.toml
+++ b/ndk-examples/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 publish = false
 
 [target.'cfg(target_os = "android")'.dependencies]
-jni = "0.19"
+jni = "0.20"
 libc = "0.2"
 log = "0.4.14"
 ndk = { path = "../ndk", features = ["api-level-23"] }

--- a/ndk-examples/examples/jni_audio.rs
+++ b/ndk-examples/examples/jni_audio.rs
@@ -1,3 +1,5 @@
+use jni::objects::JObject;
+
 #[cfg_attr(target_os = "android", ndk_glue::main(backtrace = "on"))]
 fn main() {
     enumerate_audio_devices().unwrap();
@@ -9,6 +11,7 @@ fn enumerate_audio_devices() -> Result<(), Box<dyn std::error::Error>> {
     // Create a VM for executing Java calls
     let ctx = ndk_context::android_context();
     let vm = unsafe { jni::JavaVM::from_raw(ctx.vm().cast()) }?;
+    let context = unsafe { JObject::from_raw(ctx.context().cast()) };
     let env = vm.attach_current_thread()?;
 
     // Query the global Audio Service
@@ -17,7 +20,7 @@ fn enumerate_audio_devices() -> Result<(), Box<dyn std::error::Error>> {
 
     let audio_manager = env
         .call_method(
-            ctx.context().cast(),
+            context,
             "getSystemService",
             // JNI type signature needs to be derived from the Java API
             // (ArgTys)ResultTy
@@ -36,7 +39,7 @@ fn enumerate_audio_devices() -> Result<(), Box<dyn std::error::Error>> {
 
     println!("-- Output Audio Devices --");
 
-    let device_array = devices.l()?.into_inner();
+    let device_array = devices.l()?.into_raw();
     let len = env.get_array_length(device_array)?;
     for i in 0..len {
         let device = env.get_object_array_element(device_array, i)?;
@@ -56,7 +59,7 @@ fn enumerate_audio_devices() -> Result<(), Box<dyn std::error::Error>> {
             let sample_array = env
                 .call_method(device, "getSampleRates", "()[I", &[])?
                 .l()?
-                .into_inner();
+                .into_raw();
             let len = env.get_array_length(sample_array)?;
 
             let mut sample_rates = vec![0; len as usize];

